_scripts/instruction-widget/install.js

```diff
@@ -94,26 +96,31 @@ module.exports = function(context) {
   debian_install = function() {
     template = "debian";
 
-    if (context.distro == "ubuntu" && context.version == 16.04) {
-      context.above_4 = false;
```

Comment:
With this removed, `context.above_4` is always true, so you can remove the variable entirely and update the template.